glean_parser/schemas/metrics.2-0-0.schema.yaml

```diff
@@ -348,6 +348,12 @@ definitions:
           properties:
             description:
               type: string
+            type:
```

Comment:
Just to double check: this does not require a schema bump, right?

Reply:
It should not, because it's optional!